docs/get-started/install.md

```diff
@@ -2,8 +2,65 @@
 title: 'Install'
 ---
 
-Test content
+Use the Storybook CLI to install it in a single command. Run this inside your existing project’s root directory:
 
-![optimize](./optimize.gif)
+```sh
+npx sb init
+```
+<div style="background-color:#F8FAFC">
+TODO: vet this per feedback (https://github.com/storybookjs/storybook/pull/11632#discussion_r458498081)
 
-## Heading
+</div>
+
+Storybook needs to be installed into a project that is already configured. For instance with:
+
+- 📦 [Create React App](https://reactjs.org/docs/create-a-new-react-app.html)
+- 📦 [Vue Cli](https://cli.vuejs.org/)
+
+
+Or any other tooling available. 
+
+During its install process, Storybook will look into your project's dependencies and provide you with the best configuration available.
```

Comment:
This does not match the Google doc. Readers will get lost here because we're offering them way too many choices. All of this manual config `--type` should be hidden away in the Troubleshooting section.

Reply:
Fixed with the latest commits. Let me know if you're ok with it.